### ISSUE TITLE
Getting error unable to recognize the username, password, tag

### ISSUE THREAD

**Issue by basavarajb** [keel-hq/keel#562]
when i run below command im getting error.  Attached the error image
kubectl apply -f https://sunstone.dev/keel?namespace=default&username=admin&password=admin&tag=latest

![image](https://user-images.githubusercontent.com/9286143/101982459-0ca98b00-3c9a-11eb-86ec-52061deca72a.png)

**Comment by rusenask** [comment]
I think it might be due to the OS as you are running on windows? You can also open this https://sunstone.dev/keel?namespace=default&username=admin&password=admin&tag=latest in your browser and just copy paste to a file :)

**Comment by basavarajb** [comment]
After deploying the keel pod with webrelay tunnel and secrets, keel UI prompting me again and again to signin 
![image](https://user-images.githubusercontent.com/9286143/101989388-f9acb000-3cc5-11eb-8be0-5d34ce6ce415.png)

**Comment by basavarajb** [comment]
Unable to create the web hook using custom CRD

- Enable Webhook Relay forwarding (namespace keel) and added webhook relay secret and key while executing command
-Created the simple CR (namespace keel)
-Then deployed the keel  kubectl apply -f https://sunstone.dev/keel?namespace=keel&username=admin&password=admin&tag=latest


![image](https://user-images.githubusercontent.com/9286143/102011931-64fd8d00-3d6d-11eb-989a-287b7c62dfcb.png)

**Comment by rusenask** [comment]
what's that pop up? looks like basic auth?

**Comment by basavarajb** [comment]
yes, i'm using the basic auth. It is prompting me infinite times

**Comment by rusenask** [comment]
keel already has authentication on the web ui, when you are using basic auth, it will be stripping the Authorization header that keel's UI/API is using for authentication so it just can't reach it.

**Comment by basavarajb** [comment]
I deployed the script generated from below URL
https://sunstone.dev/keel?namespace=default&username=admin&password=admin&tag=latest

**Comment by rusenask** [comment]
that's fine on keel's side, but you seem to be getting authentication request from your proxy

**Comment by basavarajb** [comment]
i exposed through webhookrelay tunnel

**Comment by rusenask** [comment]
check whether there's a basic auth enabled on the tunnel, if there is - disable it

**Comment by basavarajb** [comment]
@rusenask now UI working as expected after removing the basic auth from webhookrelay
Thanks
